lib/V8/v8-utils.cpp

```diff
@@ -307,8 +307,9 @@ static void JS_Options(v8::FunctionCallbackInfo<v8::Value> const& args) {
     TRI_V8_THROW_EXCEPTION_USAGE("options()");
   }
 
-  VPackBuilder builder =
-      ApplicationServer::server->options({"server.password"});
+  VPackBuilder builder = ApplicationServer::server->options(
+      {"server.password", "database.password", "ldap.bindpasswd",
```

Comment:
Does an option `database.password` actually exist?

Reply:
Yes:

--database.password <string>                                initial password of root user (default: "")